environments/staging.yaml

```diff
@@ -3,4 +3,10 @@ continuumLogin:
 
 continuumAdaptor:
   port: 3001
-  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
\ No newline at end of file
+  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
+
+submission:
+  scienceBeam:
+    apiUrl: "https://sciencebeam-texture.elifesciences.org/api/convert"
```

Comment:
note that this is not super stable at the moment https://github.com/elifesciences/sciencebeam-texture-formula/pull/14 keep an eye out if it breaks